postgresql/resource_postgresql_grant_test.go

```diff
@@ -2,12 +2,130 @@ package postgresql
 
 import (
 	"fmt"
+	"reflect"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/lib/pq"
 )
 
+func TestCreateGrantQuery(t *testing.T) {
+	var databaseName string = "foo"
+	var roleName string = "bar"
+
+	cases := []struct {
+		resource   *schema.ResourceData
+		privileges []string
+		expected   string
+	}{
+		{
+			resource: schema.TestResourceDataRaw(t, resourcePostgreSQLGrant().Schema, map[string]interface{}{
+				"object_type": "table",
+				"schema":      databaseName,
+				"role":        roleName,
+			}),
+			privileges: []string{"SELECT"},
+			expected:   fmt.Sprintf("GRANT SELECT ON ALL TABLES IN SCHEMA %s TO %s", pq.QuoteIdentifier(databaseName), pq.QuoteIdentifier(roleName)),
+		},
+		{
```

Comment:
Could you just add a case with multiple privileges and with grant option for table and/or sequence?